src/rev1_reply_comments.tex

```diff
@@ -0,0 +1,345 @@
+              reduce contacts. These are active throughout the period of analysis and
+              they vary with the strictness of measure. In order to do so, it is crucial
+              to take the type of contact into account; see for example our analysis of
+              potential school closures on Page~\pageref{ref.2.a}. Closed schools mean
+              that children have far fewer and parents have somewhat fewer contacts than
+              without school closures.
+
+              Finally, the very good fit of our model to official infection rates across
+              age groups increases our confidence that we manage to match actual meeting
+              patterns. If we were far off for, for example, the number of work
+              contacts, we would not be able to match either the infections of
+              individuals during working age (35-59) or of those that mostly do not work
+              anymore (60-79). 
+            }
+    \item They state that susceptibility is dependent on age. This requires references
+          and needs to be further explained. Also, younger populations are highly susceptible,
+          they may just not contribute transmission as much. On that same note, infectiousness
+          may not be independent of age
+
+          Edward Goldstein, Marc Lipsitch, Muge Cevik, On the Effect of Age on the
+          Transmission of SARS-CoV-2 in Households, Schools, and the Community, The
+          Journal of Infectious Diseases, Volume 223, Issue 3, 1 February 2021, Pages
+          362–369, \url{https://doi.org/10.1093/infdis/jiaa691}
+
+          \response{%
+              Many thanks for pointing this out, indeed we had not cited our key
+              reference for susceptibilities by age group \citep{Davies2020} in the main
+              text. We do so now on Page~\ref{r.2.b}.
+
+              We base our assumption on infectiousness being independent of age on
+              \citet{Jones2021}. That paper reports only slight differences in viral
+              loads between German adults and children: ``The least infectious youngest
+              children have 78\% (61, 94) of the peak culture probability of adults aged
+              45 to 55''. Furthermore, they caution that these lower numbers may be due
+              to different sample taking---such as the usage of smaller pediatric swab
+              devices---rather than actual lower viral loads and conclude that the
+              differences, if existent, are likely not clinically relevant.
+
+                  {\bf \color{red} ToDo: Check paper suggestion
+                      \url{https://doi.org/10.1093/infdis/jiaa691} -- are they actually checking
+                      infectiousness conditional on contact type? Or could we even reproduce
+                      what they are doing (not suggesting we should, certainly no need to spend
+                      time on it)}
```

Comment:
- It is a meta study. Some of the reviewed studies only look at infections that were attributed to household contacts in an effort to control for age differences in contact patterns. But they do not provide infectiousness patterns across all contact types we have. 
- We chose the study we chose because it was the only one that tried to really differentiate between age differences in susceptibility (given a risk contact) vs age differences in contact patterns. Moreover, they had fine grained age bins. 
- The mentioned paper confirms that susceptibility is age dependent: "We found evidence that compared to younger/middle-aged adults, children younger than 10 years have significantly lower estimated susceptibility to SARS-CoV-2 infection, while adults older than 60 years have elevated susceptibility to infection"
- The evidence for age dependent infectiousness is much weaker and not statistically significant: "There is limited evidence in the literature regarding age-related differences in infectivity, although point estimates in several studies suggest that infectivity may increase somewhat with age"

So in general I would say we are in line with the results from that paper. Given that we capture some of this difference by estimating infection probabilities that only affect adults (work) or only children (school, ...) I think we are safe to ignore the rest.

Reply:
Addressed in b6a511d